types/coin.go

```diff
@@ -592,7 +592,16 @@ var (
 // ValidateDenom validates a denomination string returning an error if it is
 // invalid.
 func ValidateDenom(denom string) error {
-	if !reDnm.MatchString(denom) {
+	var match bool
+	switch strings.Contains(denom, "/") {
+	case true:
+		// TODO: validate IBC denom prefix
+		match = true
+	default:
+		match = reDnm.MatchString(denom)
+	}
+
+	if !match {
```

Comment:
we can revert this and add a TODO on the ICS20 tests to add tests for denoms with special chars

Reply:
gonna fix all the tests first with this still there so I can ensure they pass, then will revert and comment all the tests